README.md

```diff
@@ -9,136 +9,78 @@
+[Haystack](https://haystack.deepset.ai/) is an end-to-end LLM framework that allows you to build applications powered by
+LLMs, Transformer models, vector search and more. Whether you want to perform retrieval-augmented generation (RAG),
+documentation search, question answering or answer generation, Haystack can orchestrate state-of-the-art embedding models
+and LLMs into pipelines to build end-to-end NLP applications and solve your use case.
 
-[Haystack](https://haystack.deepset.ai/) is an end-to-end LLM framework that enables you to build applications powered by LLMs, Transformer models, vector search and more. Whether you want to perform retrieval-augmented generation (RAG), documentation search, question answering or answer generation, you can use state-of-the-art embedding models and LLMs with Haystack to build end-to-end NLP applications to solve your use case.
+## Installation
 
-
-## Quickstart
-
-Haystack is built around the concept of pipelines. A pipeline is a powerful structure that performs an NLP task. It's made up of components connected together. For example, you can connect a [retriever](https://docs.haystack.deepset.ai/v2.0/docs/retrievers) and a [generator](https://docs.haystack.deepset.ai/v2.0/docs/generators) to build a Generative Question Answering pipeline that uses your own data.
-
-First, run the minimal Haystack installation:
+The simplest way to get Haystack is via pip:
 
 ```sh
 pip install haystack-ai
 ```
-👉 To build a minimal RAG pipeline that uses GPT-4 on your own data, use the [RAG Pipeline Recipe](https://docs.haystack.deepset.ai/v2.0/docs/creating-pipelines#example)
 
-## Core Concepts
+Haystack supports multiple installation methods including Docker images. For a comprehensive guide please refer
+to the [documentation](https://docs.haystack.deepset.ai/v2.0/docs/installation).
 
-⚛️ **[Components](https://docs.haystack.deepset.ai/v2.0/docs/components):** Each Component achieves one thing. Such as preprocessing documents, retrieving documents, using specific language models to answer questions, and so on. Components can `.connect()` to each other to form a complete pipeline.
+## Documentation
 
-🏃‍♀️ **[Pipelines](https://docs.haystack.deepset.ai/v2.0/docs/pipelines):** This is the standard Haystack structure that builds on top of your data to perform various NLP tasks such as retrieval augmented generation, question answering and more. Pipelines in Haystack are Directed Multigraphs composed of components. Components can receive inputs from other components and produce outputs that can be forwarded to other components.
+If you're new to the project, check out [What is Haystack?](https://haystack.deepset.ai/overview/intro) then go
```

Comment:
remove the link to the what is haystack page for now. There's a chance it won't be updated on time